### ISSUE TITLE
Writing in the console window can be blocked by "sleeping"

### ISSUE THREAD

**Issue by Jmgr** [Jmgr/actiona#46]
Writing in the console using an action or Console.print can be postponed by "pausing" the execution, for instance with Execution.pause. It should be displayed immediately instead.

Forum topic: https://jmgr.net/forum/viewtopic.php?f=30&t=1372 (French)
